package.json

```diff
@@ -26,7 +26,7 @@
     "typescript": "^2.4.2"
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "npm i && lerna bootstrap",
```

Comment:
Does it work on Windows?

Reply:
I haven't tested it, but it should be OS agnostic afaik.

Reply:
I believe `&&` works on windows well, see e.g. https://stackoverflow.com/a/8055430/69868

>     && [...]  command1 && command2
> Use to run the command following && only if the command preceding the symbol is successful. Cmd.exe runs the first command, and then runs the second command only if the first command completed successfully.